forge-gui/res/cardsfolder/upcoming/stall_for_time.txt

```diff
@@ -0,0 +1,10 @@
+Name:Stall for Time
+ManaCost:2 W
+Types:Instant
+K:Kicker:1 U
+A:SP$ Tap | TgtPrompt$ Select up to two target creatures | ValidTgts$ Creature | TargetMin$ 0 | TargetMax$ 2 | SubAbility$ DBDraw | SpellDescription$ Tap up to two target creatures. If this spell was kicked, put a stun counter on each of those creatures. (If a permanent with a stun counter on it would become untapped, remove one from it instead.)
+SVar:DBDraw:DB$ Draw  | SubAbility$ DBCounter | SpellDescription$ Draw a card.
```

Comment:
Also, should the draw come after the PutCounter here?

Reply:
Draw needs to be after the Counter